main.tf

```diff
@@ -4,7 +4,7 @@ provider "aws" {
 }
 
 module "eks-jx" {
-  source               = "jenkins-x/eks-jx/aws?ref=v1.15.12"
+  source               = "jenkins-x/eks-jx/aws"
```

Comment:
So even if you remove it, when a new eks-jx module version appears, it will append it sadly. I will try and figure out why the auto generated PRs do that ...